src/app/editor/report-issue/report-issue-modal/report-issue-modal.component.html

```diff
@@ -0,0 +1,13 @@
+<div mat-dialog-content>
+  <div class="header layout-row layout-align-start-center">
+    <h1>
+      Report Issue
+      <small>Thank you for taking the time to report an issue. When you click the the Report Issue button, Microsoft will send information about your local development environment to assist with troubleshooting.</small>
```

Comment:
Richard, can you take a look at this text and let me know if it is what you had in mind?

Reply:
This is fine